### ISSUE TITLE
Elemental Status Effects

### ISSUE THREAD

**Issue by LockerM** [DeckofAdventures/TheGame#46]
We should have status effects that correlate to the main elements as someone is playing to have easy reference. "Entangled" is something general, but "Frozen" is something specific. 

Not sure if we need to have all elements represented, or if we just make sure they all have a meaningful effect represented in the game. 

Fire: *Burned* - At the start of the affected target's turn, make a DR 3 Vitality check to see if the effect ends. On a failure, take 1 point of fire damage (bypassing AP). 

Water: *Frozen* - Movement speed is reduced by half and can only take one action on your next turn. At the end of the targets turn you can make a DR 3 Strength check to see if the effect ends. 

Air: *Stunned* - We already have this effect listed, and I think it works nicely for Lightning effects to have a Stun component.  
- *Also, not sure this is a status effect, but I think having Air have the ability to create a gust of wind forward and push targets back and creating some sort of mechanical damage effect if they slam into a surface/fall from a height. We don't have fall damage listed anywhere, might be important to just have it be 1 damage every 10 feet (example).*

Earth: Maybe we change "Entangled" to "Rooted" and leave the effect in place. Or just have Earth be connected to Entangle, not sure if the language matters as much here. 

Basically just wanting to make sure if a player takes a specific element that we have hard coded rules in place for what happens on elemental effects. The main TTRPG doesn't have a "frozen" status condition for example even though so many things do cold damage.
